### PR TITLE
Add a new command, campaign_check

### PR DESCRIPTION
**Pull request by lbonn**
Accessible from the CLI + the "new" exposed api

### CODE REVIEW

File: src/libaktualizr/campaign/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+set(SOURCES campaign.cc)
+set(HEADERS campaign.h)
+
+add_library(campaign OBJECT ${SOURCES})
+
+add_aktualizr_test(NAME campaign SOURCES campaign_test.cc ARGS ${CMAKE_CURRENT_SOURCE_DIR}/test)
```
Comment (patriotyk) [inline src/libaktualizr/campaign/CMakeLists.txt]: test -> tests? But if it works with typo, you don't need it

Comment (lbonn) [reply src/libaktualizr/campaign/CMakeLists.txt]: Which one? `${CMAKE_CURRENT_SOURCE_DIR}/test`?

It's to get the path of `https://github.com/advancedtelematic/aktualizr/blob/7acdc4c6c90098e055dad96abed4cf3dc385dbbf/src/libaktualizr/campaign/test/campaigns_sample.json`

File: src/libaktualizr/campaign/campaign.cc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        if (!install_message.empty()) {
+          LOG_ERROR << "Only one install message allowed";
+          throw;
+        }
+        install_message = o["value"].asString();
+      }
+    }
+
+    return {name, install_message};
+  } catch (std::runtime_error &exc) {
```
Comment (patriotyk) [inline src/libaktualizr/campaign/campaign.cc]: should be const reference

File: src/libaktualizr/campaign/CMakeLists.txt (hunk 1 above)
Comment (patriotyk) [reply src/libaktualizr/campaign/CMakeLists.txt]: Sorry I thought it is project source directory

File: src/libaktualizr/campaign/campaign.cc [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    LOG_ERROR << exc.what();
+    throw CampaignParseError();
+  } catch (...) {
+    throw CampaignParseError();
+  }
+}
+
+std::vector<Campaign> campaignsFromJson(Json::Value &json) {
```
Comment (patriotyk) [inline src/libaktualizr/campaign/campaign.cc]: Why json is not a const?

**Comment by lbonn** [comment]
Ok, was tested properly against backend

**Pull request merged**